main.nf

```diff
@@ -574,17 +797,32 @@ process reviewer {
 	output:
 		file("*svg")
 		//set group, file("${group}_rev.INFO") into reviewer_INFO
-
-    shell:
-    '''
-    grep LocusId !{params.expansionhunter_catalog} | sed 's/[",^ ]//g' | cut -d':' -f2 | perl -na -e 'chomp; \
-    system("REViewer --reads !{bam} \
-    --vcf !{vcf} \
-    --reference !{genome_file} \
-    --catalog !{params.expansionhunter_catalog} \
-    --locus $_ \
-    --output-prefix !{id}");'
-	'''
+		set group, file("*versions.yml") into ch_reviewer_versions
+
+	shell:
+		version_str = reviewer_version(task)
+		'''
+		grep LocusId !{params.expansionhunter_catalog} | sed 's/[",^ ]//g' | cut -d':' -f2 | perl -na -e 'chomp; \
+		system("REViewer --reads !{bam} \
+		--vcf !{vcf} \
+		--reference !{genome_file} \
+		--catalog !{params.expansionhunter_catalog} \
+		--locus $_ \
+		--output-prefix !{id}");'
+
+		echo "!{version_str}" > "!{task.process}_versions.yml"
+		'''
+
+	stub:
+		version_str = reviewer_version(task)
+		"""
+		touch ${id}.svg
+		echo ${version_str} > "${task.process}_versions.yml"
```

Comment:
```suggestion
		echo "${version_str}" > "${task.process}_versions.yml"
```

Quoting the `version_str` var in the stub fixes the issue in my last comment.

Reply:
OK, interesting, great that you spotted it

Reply:
I have heard a saying that "you should never use bash variables unquoted". Seems I was bitten by that here